packages/explat-client/src/internal/experiment-assignment-store.ts

```diff
@@ -4,46 +4,81 @@
 import type { ExperimentAssignment } from '../types';
 import * as Validations from './validations';
 
+const localStorage =
+	typeof window !== 'undefined' && window.localStorage
+		? window.localStorage
+		: // LocalStorage polyfill from https://gist.github.com/juliocesar/926500
+		  {
+				_data: {} as Record< string, string >,
+				setItem: function ( id: string, val: string ) {
+					return ( this._data[ id ] = String( val ) );
+				},
+				getItem: function ( id: string ) {
+					return this._data.hasOwnProperty( id ) ? this._data[ id ] : undefined;
+				},
+				removeItem: function ( id: string ) {
+					return delete this._data[ id ];
+				},
+				clear: function () {
+					return ( this._data = {} );
+				},
```

Comment:
As above, the return values should be undefined: https://developer.mozilla.org/en-US/docs/Web/API/Storage

```suggestion
				removeItem: function ( id: string ) {
					delete this._data[ id ];
				},
				clear: function () {
					this._data = {};
				},
```

We can also consider removing `removeItem()` until we need it, but there's no harm in keeping it for completeness.

Reply:
Yeah I think it is better to keep it